app/models/user.rb

```diff
@@ -19,4 +21,5 @@ class User < ApplicationRecord
   def downcase_email
     email.downcase!
   end
+  before_save { email.downcase! }
```

Comment:
```suggestion
  before_save :downcase_email
```and move it above downcase_email function